pages/search.js

```diff
@@ -88,8 +89,23 @@ const ErrorBox = ({ error }) => {
 
   return (
     <div>
-      <Heading h={2}>Error</Heading>
-      <p>{formatError(error)}</p>
+      <Flex justifyContent='center'>
+        <Box width={[1, 1/4]} my={5}>
+          <OONI404 />
+        </Box>
+      </Flex>
+      <Flex justifyContent='center'>
+        <Box>
+          <Heading h={4} my={4}>
+            We encountered an error. Please try again or change the filters to get different results.
+          </Heading>
+          <Box p={[1, 3]} bg='gray3'>
+            <pre>
+              {JSON.stringify(error, null, '  ')}
```

Comment:
We don't actually need the full error object, we just need `error.response.data`.

Reply:
Perhaps it's useful to also include the statusCode returned by the API backend. 

So something like:

```
{
JSON.stringify(
{
"data": error.response.data,
"statusCode": error.response.statusCode
}
, null, '  ')
}
```

Reply:
Shows this now
![image](https://user-images.githubusercontent.com/700829/65622219-38f0c300-df93-11e9-85eb-a590b832b1cc.png)